### PR TITLE
Create a sample for Language Model on One Billion Word Benchmark

### PR DESCRIPTION
**Pull request by lyqgo**
Signed-off-by: yqliu <yiqing.liu@intel.com>

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
First of all, thanks for your intention to contribute to Open Model Zoo. But, could you please review Open Model Zoo [contribution guide](https://github.com/opencv/open_model_zoo/blob/develop/CONTRIBUTING.md) to see what is expected from contribution to OMZ?

File: demos/python_demos/language_model_demo/lm_1b_sample.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
```
Comment (eizamaliev) [inline demos/python_demos/language_model_demo/lm_1b_sample.py]: Please rename `lm_1b_sample.py` to `language_model_demo.py`
Also please read [contribution guide](https://github.com/opencv/open_model_zoo/blob/develop/CONTRIBUTING.md#demo)

File: demos/python_demos/language_model_demo/language_model_demo.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from __future__ import print_function
+import sys
+import os
+from argparse import ArgumentParser, SUPPRESS
+import cv2
```
Comment (eaidova) [inline demos/python_demos/language_model_demo/language_model_demo.py]: there is no usage opencv

File: demos/python_demos/language_model_demo/language_model_demo.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+import numpy as np
+import logging as log
+from time import time
+from openvino.inference_engine import IENetwork, IECore
+import data_utils
+from six.moves import xrange
+import tensorflow as tf
```
Comment (eaidova) [inline demos/python_demos/language_model_demo/language_model_demo.py]: there is no usage tensorflow

File: demos/python_demos/language_model_demo/language_model_demo.py [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+from __future__ import print_function
+import sys
+import os
+from argparse import ArgumentParser, SUPPRESS
+import cv2
+import numpy as np
+import logging as log
+from time import time
+from openvino.inference_engine import IENetwork, IECore
+import data_utils
```
Comment (eaidova) [inline demos/python_demos/language_model_demo/language_model_demo.py]: how to get this module? I do not see that `CharsVocabulary` provided with demo

**Pull request closed without merging**